### ISSUE TITLE
After the reboot, loss of all configurations

### ISSUE THREAD

**Issue by LazyCuteLion** [SpazeDog/xposed-additions#29]
After the reboot, loss of all configurations
my device is oneplus 3.

**Comment by token0** [comment]
Same on Z5 Compact, but this is about Nougat and not about phone model.

**Comment by francwalter** [comment]
Same issue on a Cubot Note Plus with Android 7.0 (Nougat) and Version 3.6.3 from 21.12.2015 with the bought Pro Version from Google Play.
Error Log stays empty.
Is there mayebe at least a way to restore it through some script (by Tasker etc.)?
Titanium Restore doesnt restore the configured buttons settings.

**Comment by AlexFundorin** [comment]
I see many requests around the internet, asking to fix this issue with Nougat. Since it's open source, maybe someone could make a fork and fix this for us?

**Comment by francwalter** [comment]
Seams that is not at all that easy. Asked  [rueolps](https://forum.xda-developers.com/showpost.php?p=74377722&postcount=2752) on the xda Thread about it, he tried a lot and it is too difficult to just adjust a bit to make it work again.
I guess this is about better to forget :(

**Comment by sven-cnrd** [comment]
To anyone still waiting: there are builds that address this problem, see https://forum.xda-developers.com/xposed/modules/mod-xposed-additions-t2294274/page278

**Comment by francwalter** [comment]
Tested that "3.6.4" version with no success. Apart from not working it is not anymore a Pro-Version, as that version doesn't recognize the Pro-Key (bought on Google Play).
So unfortunately not the solution till yet 👎

**Comment by sven-cnrd** [comment]
3.6.4 should keep the settings, 3.6.5 also skips the pro check. Don't forget to reboot after isntallation, since the changes are within the xposed module.

**Comment by francwalter** [comment]
Yes, should keep them, but doesn't keep (on my phone). Without Pro Features the app is half useless (for me).
Several reboots I tried, that clear.

**Comment by AlexFundorin** [comment]
It does save settings. Just install 3.6.5 (with pro unlocked), add some settings to it, reboot and wait around 30 seconds. There would be no saved settings right after reboot, but they'll reappear after a while.

**Comment by francwalter** [comment]
OHHH!
Yes!!!
That 3.6.5 works 👍 👍 👍 
Saves settings and is the pro version.
Thank!!!

**Comment by EugenNY** [comment]
you can find xposedAdditions_3.6.5.apk here https://github.com/sven-cnrd/xposed-additions/releases/

and here is the fixed xda dev link: https://forum.xda-developers.com/t/mod-xposed-2-3-2015-04-09-xposed-additions.2294274/page-142
